src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -41,9 +63,10 @@ private QuantityUtil() {
    * @param b Second quantity to compare
    * @param absQuantityTolerance Permissible absolute tolerance
    * @param <Q> Type of Quantity
-   * @return true, if both quantities' values differ less then the given tolerance else false
+   * @return true, if both quantities' values differ less then the given tolerance else false (based
+   *     on a's unit)
    */
-  public static <Q extends Quantity<Q>> boolean considerablyAbsEqual(
+  public static <Q extends Quantity<Q>> boolean isAbsolutelyEquivalent(
```

Comment:
Please keep the old implementation `considerablyAbsEqual`, mark it as deprecated and call the new function instead. This could really help us not breaking too much.

Reply:
Sorry for this. Just realized, that it was "former me", that suggested renaming. 😞